app/components/floating_input/floating_input_container.tsx

```
@@ -194,9 +199,10 @@ const FloatingInputContainer = ({
     return (
         <TouchableWithoutFeedback
             onLayout={onLayout}
+            accessible={isRunningE2e ? false : undefined}
         >
             <View style={styles.container}>
-                <Pressable onPress={handlePressOnContainer}>
+                <Pressable onPress={handlePressOnContainer} accessible={isRunningE2e ? false : undefined}>
```

ESLint

[error] Prop `accessible` must be placed on a new line

(react/jsx-max-props-per-line)